wifiphisher/pywifiphisher.py

```diff
@@ -778,12 +781,19 @@ def start(self):
             # set the channel on the deauthenticating interface
             mon_iface.set_channel(int(channel))
 
-            # start deauthenticating all client on target access point
-            deauthentication = deauth.Deauthentication(ap_mac,
-                                                       mon_iface.get_name())
-            if args.lure10_exploit:
-                deauthentication.add_lure10_beacons(LOCS_DIR + args.lure10_exploit)
-            deauthentication.deauthenticate()
+            # Start Extension Manager
+            shared_data = {
+                'target_ap_channel': channel or "",
+                'target_ap_essid': essid or "",
+                'target_ap_bssid': ap_mac or "",
+                'target_ap_encryption': enctype or "",
+                'target_ap_logo_path': ap_logo_path or "",
+                'rogue_ap_mac': ap_mac,
+                'APs': APs_context, 
+                'args': args
+            }
```

Comment:
Yes, `namedtuple` may turn out better.

The plan is to have a `shared_data` that will be shared among all extensions *and* templates. Currently, we feed templates with kinda different data but should be the same as in the extensions.